lib/tenstorrent/bh_arc/gddr.c

```diff
@@ -238,6 +241,25 @@ int CheckHwMemtestResult(uint8_t gddr_inst, k_timepoint_t timeout)
 	return 0;
 }
 
+void wipe_mrisc_l1(uint8_t tensix_x, uint8_t tensix_y)
```

Comment:
```suggestion
static void wipe_mrisc_l1(uint8_t tensix_x, uint8_t tensix_y)
```

Reply:
Static, since it should be encapsulated in the code that deals with MRISC, not adding more spaghetti called from somewhere else.

I.e. when MRISC is initialized, wipe the contents of l1 _for MRISC_.